torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -734,6 +737,100 @@ llvm::Error PyTorchModelLoader::loadMin(const torch::jit::Node *ptNode) {
   return addGlowNodeValue(outputs[0], output);
 }
 
+llvm::Error PyTorchModelLoader::loadTopK(const torch::jit::Node *ptNode) {
```

Comment:
Since we can't fuse topK (right?) maybe we should wait on this since we can't test it yet.